docs/Reference/Rest_API/API-Objects.md

```diff
@@ -0,0 +1,94 @@
+| **deposit_root**     | Bytes32             | Hash tree root of all deposits in a deposit contract. |
+| **deposit_count**    | uint64              | Number of successful validator deposits into the contract.|
+| **block_hash**       | Bytes32             | ETH1 block hash that contains the deposit root. |
+
+## Attestation
+
+Attestation information for the block.
+
+!!! note
+    Produces a blank `signature` field for the 
+    [`/validator/attestation`](https://pegasyseng.github.io/teku/stable/#operation/getValidatorAttestation)
+    endpoint, which the validator later signs.
+
+| Field                | Type                | Description                                         |
+|----------------------|:-------------------:|-----------------------------------------------------|
+| **aggregation_bits** | Bitlist             | List of bits containing a single bit for each member of the committee. |
+| **data**             | Object              | [Attestation data](#attestation-data) signed by the validator (or collection of validators). |
+| **signature**        | Bytes96             | BLS signature of the attestation.                   |
+
+### Attestation Data
+
+| Field                | Type                | Description                                         |
+|----------------------|:-------------------:|-----------------------------------------------------|
+| **slot**             | uint32              | Slot that the validator or committee is assigned to attest. |
+| **index**            | uint32              | Index of the committee making  the attestation.     |
+| **beacon_block_root** | Bytes32            | Block root of the beacon block for the assigned slot. |
+| **source**           | Object              | Checkpoint (`epoch` and `root`) during the assigned slot. |
+| **target**           | Object              | The checkpoint (`epoch` and `root`) attesters are attempting to justify. |
+
+## Validators
+
+A list of validator information returned by
+[`/beacon/validators`](https://pegasyseng.github.io/teku/stable/#operation/getBeaconValidators).
+
+| Field                | Type                | Description                                         |
+|----------------------|:-------------------:|-----------------------------------------------------|
+| **pubkey**           | Bytes48             | Validator's public key.                             |
+| **validator_index**  | uint32              | Validator index within the beacon state.            |
+| **balance**          | uint32              | Account balance in Gwei.                            |
+| **validator**        | Object              | [Object containing validator information](#validator). |
+| **total_size**       | uint32              | Total number of validators.                         |
+| **next_page_token**  | uint32              | Next page number of results.                        |
```

Comment:
Validator Index is integer (signed 32 bit value)
the others marked uint32 should be uint64

Reply:
Done